demo-ui/app/uk/gov/ons/addressIndex/demoui/model/SingleSearchForm.scala

```diff
@@ -18,13 +18,15 @@ case class Address(uprn: String,
                    matchScore: Float,
                    fullAddress: String)
 
-case class SingleSearchForm(address: Option[String])
-
-case class Candidate(matchScore: Int, matchingAddress: Address)
-
-case class BulkMatchResponse(matchFound: Option[Int], possibleMatches: Option[Int], noMatch: Option[Int], totalNumberOfAddresses: Option[Int])
+case class SingleSearchForm(
+ address: String,
+ format: String = "paf"
+)
+object SingleSearchForm {
+  implicit val jsonFmt = Json.format[SingleSearchForm]
+}
 
 object JSONImplicits {
```

Comment:
please move to companion objects and use `Json.format[MyTypeHere]`, It'll give you reads and writes :)